### PR TITLE
Create ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b…

### PR DESCRIPTION
**Pull request by craigimmel**
…4559.json

# Pull Request Template

## Description

Please include a short summary of the changes in this PR.

## Type of change

- [ ] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

File: mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559",
```
Comment (VladislavKudrin) [inline mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json:2]: Hi @craigimmel,

the website shows a different policyId: c7dcfa416c127f630b263c7e0fe0564430cfa9c56bba43e1a37c6915

Can you please change that on the website or we will close the submission in 2 days.

Thanks!

**Comment by craigimmel** [comment]
Ok thanks for the heads up. We are sunsetting GOKEY v1 and replacing it
with the new Policy. We'll add the new policy to the webpage and update to
reflect this change.

File: mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json (hunk 1 above)
Comment (craigimmel) [reply mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json:2]: Hey @VladislavKudrin 

Thanks for the heads up. I have updated the policy id in the website footer at https://gokey.finance. Please let me know if any additional changes are needed.

Best,
Craig
Screenshot 2024-04-06 131928

File: mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                "publicKey": "0a39954288c8bcec75c7ec374ad6dd2071bdb5df854a920572b37b4a22fa3b6a"
+            }
+        ]
+    },
+    "description": {
+        "sequenceNumber": 0,
+        "value": "GOKEY is the World's first cryptocurrency to dynamically adjust its digital scarcity at rates corresponding to changes in internationally recognized real estate price indices, and is the primary governance, payment and utility token of the GoKey real estate financial ecosystem.",
+        "signatures": [
+            {
+                "signature": "1147c861f5ac5e7c9ba9432664a960af87232ac9e88e3b4267da46ecabb8ef0bd150a05cbb2d888f83ee5b668a8683a9f615eb720ee97faa3d6a5dc9d85aa806",
```
Comment (VladislavKudrin) [inline mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json:59]: @craigimmel,

signature for the description seems to be wrong. Can you please resign the file and update the PR?

**Pull request closed without merging**